src/sentry/static/sentry/app/views/settings/project/projectOwnership/codeowners.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import TextareaAutosize from 'react-autosize-textarea';
+import styled from '@emotion/styled';
+import {withTheme} from 'emotion-theming';
+import moment from 'moment';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
+import Tag from 'app/components/tag';
+import {IconGithub, IconGitlab} from 'app/icons';
+import {inputStyles} from 'app/styles/input';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import {Theme} from 'app/utils/theme';
+
+type Props = AsyncComponent['props'] & {
+  theme: Theme;
+  organization: Organization;
+  project: Project;
+};
+
+type State = {} & AsyncComponent['state'];
+
+class CodeOwners extends AsyncComponent<Props, State> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization, project} = this.props;
+    return [
+      [
+        'codeowners',
+        `/projects/${organization.slug}/${project.slug}/codeowners/?expand=codeMapping`,
+      ],
+    ];
+  }
+  renderIcon(provider) {
+    switch (provider) {
+      case 'github':
+        return <IconGithub size="md" />;
+      case 'gitlab':
+        return <IconGitlab size="md" />;
+      default:
+        return null;
+    }
+  }
+  renderView(data) {
+    const {theme} = this.props;
+    const {
+      raw,
+      dateUpdated,
+      provider,
+      codeMapping: {repoName},
+    } = data;
+    return (
+      <Container>
+        <RulesHeader>
+          <TitleContainer>
+            {this.renderIcon(provider)}
+            <Title>CODEOWNERS</Title>
+          </TitleContainer>
+          <ReadOnlyTag type="default">{'Read Only'}</ReadOnlyTag>
+          <Repository>{repoName}</Repository>
+          <Detail />
```

Comment:
This is just to placate the linter since I already styled it but there's no info to put in this component.